traffic_ops/app/lib/API/Job.pm

```diff
@@ -41,12 +42,27 @@ sub index {
 	my $ds_id   = $self->param('dsId');
 	my $user_id = $self->param('userId');
 
-	if ( !&is_oper($self) ) {
-		return $self->forbidden();
-	}
-
 	my %criteria;
 	if ( defined $ds_id ) {
+
+		my $ds = $self->db->resultset('Deliveryservice')->find( { id => $ds_id } );
+		if ( !defined($ds) ) {
+			return $self->not_found();
+		}
+
+		my $tenant_utils = Utils::Tenant->new($self);
+		my $tenants_data = $tenant_utils->create_tenants_data_from_db();
+
+		if ( $tenant_utils->use_tenancy) {
```

Comment:
use_tenancy is a function, you are missing the "()", unless it is not required in perl

Reply:
good catch. thanks. but i tested it so maybe perl doesn't care?